appcues/src/main/java/com/appcues/ui/modal/FullScreenModal.kt

```diff
@@ -53,6 +53,7 @@ internal fun FullScreenModal(
         val fullHeight = heightDerivedOf(windowInfo)
         val enterAnimation = enterTransitionDerivedOf(windowInfo)
         val exitAnimation = exitTransitionDerivedOf(windowInfo)
+        val isDark = isSystemInDarkTheme()
```

Comment:
really unrelated to this PR -

reminds me, meant to ask here and on iOS as well - do we respect any in-app override of the system light/dark preference? I know in iOS at least, the app can force into light or dark mode programmatically, independent of the broader system setting. it would be ideal if our experiences matched whatever the app was using.